core/primitives/src/upgrade_schedule.rs

```diff
@@ -134,6 +134,19 @@ impl ProtocolUpgradeVotingSchedule {
         result
     }
 
+    /// This method returns the protocol version that the node should vote for now.
```

Comment:
very nit. can you change "for now" to "for realtime" (just being picky since "for now" may mean other things)

Reply:
Maybe pass now as an argument and rename to something like "get_protocol_version_for_voting"?

Reply:
I changed it to `protocol_version_to_vote_for` and `protocol_version_to_vote_for_at_date`, hope that's better.